internal-import-file/import-document-ai/src/reportimporter/core.py

```diff
@@ -144,7 +144,7 @@ def _download_import_file(self, data: Dict) -> Tuple[str, BytesIO]:
 
     def _process_parsing_results(
         self, parsed: List[Dict], context_entity: Dict
-    ) -> (List[Dict], List[str]):
+    ) -> List[Dict] | List[str]:
```

Comment:
I think the type hint should be Tuple[List[Dict], List[str]], considering return observables, entities.
Additionally, I just noticed that observables is initialized twice (lines 148 and 150).

```suggestion
    ) -> Tuple[List[Dict], List[str]]: